data/json/items/gun/223.json

```diff
@@ -132,6 +132,69 @@
     ],
     "melee_damage": { "bash": 9 }
   },
+  {
+    "id": "plr16",
+    "copy-from": "rifle_semi",
+    "looks_like": "skorpion_61",
+    "type": "GUN",
+    "name": { "str": "Kel-Tec PLR16" },
+    "description": "Semi-automatic pistol, chambered in 5.56 NATO, using gas piston to reduce lenght and frame constructed from reinforced polymer to make the it both durable and light-weight. This one comes with plastic handguard.",
+    "weight": "1800 g",
+    "volume": "1665 ml",
+    "longest_side": "485 mm",
+    "barrel_length": "234 mm",
+    "price": 86900,
+    "price_postapoc": 5500,
+    "to_hit": -2,
+    "material": [ "steel", "aluminum", "plastic" ],
+    "symbol": "(",
+    "color": "dark_gray",
+    "ammo": [ "223" ],
+    "skill": "pistol",
+    "range": -6,
+    "dispersion": 300,
+    "durability": 6,
+    "min_cycle_recoil": 1350,
+    "valid_mod_locations": [
+      [ "barrel", 1 ],
+      [ "bore", 1 ],
+      [ "brass catcher", 1 ],
+      [ "grip", 1 ],
+      [ "mechanism", 4 ],
+      [ "magazine", 1 ],
+      [ "muzzle", 1 ],
+      [ "rail mount", 2 ],
+      [ "sights", 1 ],
+      [ "stock mount", 1 ],
+      [ "underbarrel mount", 1 ]
+    ],
+    "pocket_data": [
+      {
+        "pocket_type": "MAGAZINE_WELL",
+        "item_restriction": [
+          "stanag30",
+          "stanag5",
+          "stanag5ranger",
+          "stanag10",
+          "stanag10ranger",
+          "stanag20",
+          "stanag20ranger",
+          "stanag30ranger",
+          "stanag40",
+          "stanag40ranger",
+          "stanag50",
+          "stanag60",
+          "stanag60drum",
+          "stanag90",
+          "stanag100",
+          "stanag100drum",
+          "stanag150",
+          "survivor223mag"
+        ]
+      }
+    ],
+    "melee_damage": { "bash": 9 }
+  },
```

Comment:
Could you also add a generic gun variant for use when the branded gun names option isn’t toggled to true? It would save some work later on when a clean sweep of the guns are made and more such variants are included. You can see how the Remington 700 in .270 Winchester does it for an example.